documentation/troubleshooting.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Troubleshooting Kubernetes vSphere Cloud Provider
+---
+This document is aimed to help debug issues in the Kuberentes vSphere Cloud Provider.
+
+For general Kubernetes Issue, Please visit [https://kubernetes.io/docs/tasks/debug-application-cluster/debug-cluster/](https://kubernetes.io/docs/tasks/debug-application-cluster/debug-cluster/)
+
+## What logs can I inspect?
+For digging deeper and help gather data for the technical support we need to see the log files.
```

Comment:
Please remove this statement as it's redundant.

Reply:
removed.